src/src/com/microsoft/aad/adal/DefaultTokenCacheStore.java

```diff
@@ -315,7 +317,13 @@ public void clearTokensForUser(String userid) {
         for (TokenCacheItem item : results) {
             if (item.getUserInfo() != null
                     && item.getUserInfo().getUserId().equalsIgnoreCase(userid)) {
-                this.removeItem(CacheKey.createCacheKey(item));
+                try {
+                    this.removeItem(CacheKey.createCacheKey(item));
```

Comment:
why remove item started to throw exception? I don't see where is the change in removeItem method?